dmoj/cptbox/sandbox.py

```diff
@@ -227,15 +227,7 @@ def _protection_fault(self, syscall):
                     callname = by_id[id]
                     break
 
-            print>> sys.stderr, 'Protection fault on: %d (%s)' % (syscall, callname)
-            print>> sys.stderr, 'Arg0: 0x%016x' % self.debugger.uarg0
-            print>> sys.stderr, 'Arg1: 0x%016x' % self.debugger.uarg1
-            print>> sys.stderr, 'Arg2: 0x%016x' % self.debugger.uarg2
-            print>> sys.stderr, 'Arg3: 0x%016x' % self.debugger.uarg3
-            print>> sys.stderr, 'Arg4: 0x%016x' % self.debugger.uarg4
-            print>> sys.stderr, 'Arg5: 0x%016x' % self.debugger.uarg5
-
-            self.protection_fault = (syscall, callname)
+            self.protection_fault = (syscall, callname, self.debugger)
```

Comment:
Brain damaged. This will never work since the attributes are read real time. In other words, they are useless if you don't read them immediately, especially after the process terminates.